public/js/popup.js

```diff
@@ -26,6 +26,37 @@ function locationToString(location) {
     return `[${start}] - [${end}]`;
 }
 
+function getLineFromFile(code, location) {
+    const [[startLine]] = location;
+    const lines = code.split('\n');
+
+    return lines[startLine - 1];
+}
+
+async function fetchCodeLine(event, name, version, file, location) {
```

Comment:
Not sure if we need name, version and file as argument of this function. We could also pass the unpkg URL directly as a parameter.